frontend/amundsen_application/static/js/components/LineageGraph/index.tsx

```diff
@@ -0,0 +1,84 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import * as ReactDOMServer from 'react-dom/server';
+import { Lineage } from 'interfaces';
+import { LeftArrowIcon, RightArrowIcon } from '../SVGIcons';
+import d3Lineage from './wrapperLineage';
+import './styles.scss';
+
+export interface LineageGraphProps {
+  lineage: Lineage;
+}
+
+interface LineageGraphState {}
+
+export class LineageGraph extends React.Component<
```

Comment:
We want to move out of using the class syntax for new work, could you try to use React Hooks instead?

Reply:
This might be useful to you: https://github.com/britecharts/britecharts-react/blob/mi-typescript-move/src/charts/ComponentDemo.tsx